addons/web/static/src/core/ui/block_ui.scss

```diff
@@ -2,4 +2,5 @@
 	cursor: wait;
 	-webkit-backdrop-filter: blur(2px);
 	backdrop-filter: blur(2px);
+	z-index: 1100 !important;
```

Comment:
Great suggestion, thanks!!

Reply:
After discussing it with @stefanorigano, we decided to hard-code the value of the `z-index` (1070) in order to avoid further complications with the loading of the dependencies of the scss variables in `point_of_sale`. For that reason we will add a relevant note/comment.